crypto/armcap.c

```diff
@@ -169,6 +170,7 @@ static unsigned long getauxval(unsigned long key)
 #  define HWCAP_CE_SHA1          (1 << 5)
 #  define HWCAP_CE_SHA256        (1 << 6)
 #  define HWCAP_CPUID            (1 << 11)
+#  define HWCAP_SM3              (1 << 18)
```

Comment:
Is it more appropriate to use `HWCAP_CE_SM3` here?

Reply:
Yes, HWCAP_CE_SM3 is better.  I'll rename it.